vllm/config.py

```diff
@@ -1537,11 +1537,13 @@ def _verify_args(self) -> None:
         if self.use_ray:
             from vllm.executor import ray_utils
             ray_utils.assert_ray_available()
-        if current_platform.is_rocm():
+        if (current_platform.is_rocm()
+                and current_platform.get_device_capability() is not None
+                and current_platform.get_device_capability() < (9, 4)):
```

Comment:
This will fix CI for you:

```suggestion
        device_capability = current_platform.get_device_capability()
        if (current_platform.is_rocm() and device_capability is not None
                and device_capability < (9, 4)):
```